frontend/lib/home/home_page.dart

```diff
@@ -1,62 +1,111 @@
+import 'package:ehrenamtskarte/home/app_flows_stack.dart';
 import 'package:flutter/material.dart';
 
 import '../map/map_page.dart';
 import '../identification/identification_page.dart';
 import '../search/search_page.dart';
+import './app_flow.dart';
```

Comment:
Which import style do we use now? Relative or "absolute" paths? Maybe we should use the package version because the flutter plugin autoimport works that way?

Reply:
Yep, you are right. I misunderstood the style guide at first, it says you should only use the relative version if the imported file is in exactly the same directory (like in the new line 7, but not in the lines 4-6). And, as you said, the autoimporter also does it that way, so that is a good reason for us to do it that way.